src/app/server/java/src/chip/appserver/Fabric.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *   Copyright (c) 2021 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package chip.appserver;
+
+public class Fabric {
+
+  public int vendorId;
```

Comment:
you might consider adding fabric deletion methods to allow the user to delete a fabric that they don't want anymore. If you add a delete method, you will need to make sure you don't allow deletion of the TV's own fabric (the one used by the DeviceCommissioner of tv-app, eg. GetDeviceCommissioner()->GetFabricInfo())

Reply:
another helpful feature is to list the ACLs set on the TV and allow deletion of the ones associated with the TV's own fabric since each ACL corresponds to a casting client app. Use chip::Access::GetAccessControl() to get the ACL entries.